src/common/time/src/ttl.rs

```diff
@@ -75,15 +77,6 @@ impl TimeToLive {
         })
     }
 
-    /// Print TimeToLive as string
-    pub fn to_string_opt(&self) -> Option<String> {
-        match self {
-            TimeToLive::Instant => Some(INSTANT.to_string()),
-            TimeToLive::Duration(d) => Some(humantime::format_duration(*d).to_string()),
```

Comment:
Could `duration.to_string()` replace `humantime::format_duration(*d).to_string()`?

Reply:
> Could `duration.to_string()` replace `humantime::format_duration(*d).to_string()`?

` humantime::Duration` wrapper std::Duration and impl `Display`(which `std::Duration` didn't not because too many date format), but `ttl` store std::Duration for simplicity and `humantime::Duration` missing some derived traits

Reply:
Yes. So that's my actual question. If you remove `to_string_opt`, are you sure all the cases can be replaced by `ttl.map(|ttl| ttl.to_string())` ?

Reply:
> Yes. So that's my actual question. If you remove `to_string_opt`, are you sure all the cases can be replaced by `ttl.map(|ttl| ttl.to_string())` ?

impl ttl for enum like this so it should be ok?, and made sure that enum' variant is never access directly outside `ttl.rs` file:
```
impl Display for DatabaseTimeToLive {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        match self {
            DatabaseTimeToLive::Forever => write!(f, "{}", FOREVER),
            DatabaseTimeToLive::Duration(d) => write!(f, "{}", humantime::Duration::from(*d)),
        }
    }
}
```